improver/utilities/cube_manipulation.py

```diff
@@ -364,7 +240,11 @@ def process(self, cubes_in, check_time_bounds_ranges=False):
         # create copies of input cubes so as not to modify in place
         cubelist = iris.cube.CubeList([])
         for cube in cubes_in:
-            cubelist.append(cube.copy())
+            if slice_over_realization:
+                for slice in cube.slices_over("realization"):
```

Comment:
Codacy found an issue: [Redefining built-in 'slice'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5785468)